pkg/nvml/zz_generated.api.go

```diff
@@ -184,9 +184,6 @@ var (
 	DeviceGetVgpuCapabilities                       = libnvml.DeviceGetVgpuCapabilities
 	DeviceGetVgpuMetadata                           = libnvml.DeviceGetVgpuMetadata
 	DeviceGetVgpuProcessUtilization                 = libnvml.DeviceGetVgpuProcessUtilization
-	DeviceGetVgpuSchedulerCapabilities              = libnvml.DeviceGetVgpuSchedulerCapabilities
```

Comment:
Since we're generating APIs now, could we call out the removal of these functions in a changelog / release notes? (more of a nice-to-have)

Reply:
They were added back in. There were however, 2 legitimate removals:
```
-   CcuGetStreamState() (int, Return)
-   CcuSetStreamState(int) Return
```
Not sure if these were intentional...

```